tests/i18n/urls_default_unprefixed.py

```diff
@@ -3,8 +3,10 @@
 from django.http import HttpResponse
 from django.utils.translation import ugettext_lazy as _
 
+
 urlpatterns = i18n_patterns(
     url(r'^(?P<arg>[\w-]+)-page', lambda request, **arg: HttpResponse(_("Yes"))),
     url(r'^simple/$', lambda r: HttpResponse(_("Yes"))),
+    url(r'^(.+)/(.+)/$', lambda: HttpResponse("")),
```

Comment:
Reading this again, I believe you need to specify `*args` as the argument of the lambda function. If this gets called, you will end up with a `TypeError` (see below) instead of the expected `HttpResponse`.
```
TypeError: <lambda>() takes 0 positional arguments but 2 were given
```